spec/requests/portfolios_rbac_spec.rb

```diff
@@ -24,5 +25,24 @@
 
       expect(response).to have_http_status(403)
     end
+
+    context "with filtering" do
+      before do
+        allow(RBAC::Access).to receive(:new).with('portfolios', 'read').and_return(double_access_obj)
+        allow(double_access_obj).to receive(:process).and_return(double_access_obj)
+        get "#{api('1.0')}/portfolios?filter[id]=#{portfolio1.id}", :headers => default_headers
```

Comment:
@lindgrenj6 Should this be filtered on name instead of id?

Reply:
Updated to filter on name instead of id.